exasol_script_languages_container_ci_setup/cli/commands/generate_buildspec.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+from typing import Tuple, Optional
+
+import click
+
+from exasol_script_languages_container_ci_setup.cli.cli import cli
+from exasol_script_languages_container_ci_setup.lib.run_generate_buildspec import run_generate_buildspec
+
+
+@cli.command()
+@click.option('--flavor-root-path', required=True, multiple=True,
+              type=click.Path(file_okay=False, dir_okay=True, exists=True),
+              help="Path where script language container flavors are located.")
+@click.option('--output-path', type=click.Path(file_okay=False, dir_okay=True, exists=True, writable=True),
+              default="./aws-code-build/ci",
+              help="Path where buildspec files will be deployed.")
+@click.option('--config-file', type=click.Path(file_okay=True, dir_okay=False, exists=True),
+              help="Configuration file for build, project specific.")
```

Comment:
This description is grammatically a bit weird

Reply:
Done